hw/ip/otbn/dv/uvm/gen-binaries.py

```diff
@@ -61,60 +61,29 @@ def run(self, cmd: List[str]) -> None:
         subprocess.run(cmd, env=env)
 
 
-def take_env_line(dst: Dict[str, str], path: str, line_number: int,
-                  line: str) -> None:
-    '''Read one line from a .env file, updating dst.'''
-    line = line.split('#', 1)[0].strip()
-    if not line:
-        return
+def read_toolchain(otbn_dir: str) -> Toolchain:
+    '''Reads environment variables to get toolchain info.'''
+    env_dict = {}  # type: Dict[str, str]
 
-    parts = line.split('=', 1)
-    if len(parts) != 2:
-        raise RuntimeError('{}:{}: No equals sign in line {!r}.'.format(
-            path, line_number, line))
+    # OTBN assembler
+    env_dict['OTBN_AS'] = os.getenv('OTBN_AS')
+    if env_dict['OTBN_AS'] is None:
+        env_dict['OTBN_AS'] = f"{otbn_dir}/util/otbn_as.py"
```

Comment:
Do we ever want to override the path to `otbn_as.py` or `otbn_ld.py`? These are scripts from within the project, so that seems like a rather strange thing to do?

Reply:
Fixed.